tools/val/val.cpp

```diff
@@ -58,6 +58,7 @@ NOTE: The validator is a work in progress.
                                    uniform, storage buffer, and push constant layouts.  Scalar layout
                                    rules are more permissive than relaxed block layout so in effect
                                    this will override the --relax-block-layout option.
+  --workgroup-scalar-block-layout  Enable scalar block layout when checking Workgroup block layouts.
```

Comment:
This needs propagated to spirv-opt as well and spirv-opt will have to pass it along to the validator.

Reply:
Done.